tests/robot-tests/tests/libs/common.robot

```diff
@@ -423,6 +423,19 @@ user checks element is not visible
     [Arguments]    ${element}    ${wait}=${timeout}
     element should not be visible    ${element}    ${wait}
 
+user checks element is visually hidden
+    [Arguments]    ${selector}    ${parent}=css:body
+    user checks element has class
+    ...    ${selector}
+    ...    govuk-visually-hidden
+    ...    ${parent}
```

Comment:
Selenium Library has keywords [Element Should Be Visible](https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Element%20Should%20Be%20Visible) and [Wait Until Element Is Visible](https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Wait%20Until%20Element%20Is%20Visible)

Although I don't think they worked for me well in the past, so not sure it's worth exploring, but I thought I'd mention it.

Reply:
Chatted to Mark to say the footnotes are still considered visible with these keywords due to the way the `gov-uk-visually-hidden` style is hiding them.